Gems/PhysX/Code/Editor/EditorJointConfiguration.cpp

```diff
@@ -512,4 +514,37 @@ namespace PhysX
                 followerEntityName.c_str());
         }
     }
+
+    void EditorJointMotorConfig::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<EditorJointMotorConfig>()
+                    ->Version(1)
+                    ->Field("Use Motor", &EditorJointMotorConfig::m_useMotor)
+                    ->Field("Force Limit", &EditorJointMotorConfig::m_driveForceLimit)
+                    ->Field("Gear Ratio", &EditorJointMotorConfig::m_gearRatio)
```

Comment:
nit: This is super minor but I think for serialized fields (`SerializeContext` not `EditContext` we prefer no spaces), so these would be `UseMotor`, `ForceLimit` etc...

Reply:
Done, I also removed gearRatio - it is no supported by D6. 
We can reintroduce it.